apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt

```diff
@@ -0,0 +1,37 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.get
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class ListAssertionSamples {
+
+    @Test
+    fun get() {
+        val list = listOf(0, 1, 2)
+
+        expect(list).get(0).toBe(0)
+
+        fails {
+            // index is out of bound
+            expect(list).get(3).toBe(3)
```

Comment:
Please explain,as in pairAssertions , that the subject changes etc

Reply:
ok